collector.go

```diff
@@ -0,0 +1,110 @@
+package redigosrv
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gomodule/redigo/redis"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+//RedigoCollector struct to access metrics
+type RedigoCollector struct {
+	pool                PoolStats
+	subscriptionsActive prometheus.Gauge
+	publishTrafficSize  prometheus.Counter
+	subscribeSuccesses  prometheus.Counter
+	subscribeFailures   prometheus.Counter
+	commandCalls        *prometheus.CounterVec
+	methodDuration      *prometheus.CounterVec
+	activeCount         *prometheus.Desc
+	idleCount           *prometheus.Desc
```

Comment:
Sorry man, I thought I had commented on this but I did not. When you see `redigo_active_count` at the `/metrics` endpoint, what do you think it is? Confusing right? Try to put a more explanatory name. May be `redigo_%spool_active_connections` and `redigo_%spool_idle_connections`.

The same about the `Desc` names, it is not even a count. It is a gauge. So just come up with a better name too.

Reply:
Because it is a gauge? Ever that Prometheus runs Collect method a new ConstMetric will receive the current value of PoolStats and write in the channel, or not?